scripts/configure.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
+1 Although I would leave `-x` out of the scope here to keep pod logs cleaner.

Reply:
Ok. I prefer to see the commands being executed, but if it's not necessary, then fine.

Just keep in mind that some perfectly legal bash scripts need to be fixed after you add the `set`. E.g. if the script used undefined variable, expecting it will evaluate to empty string, that will no longer work now.